### PR TITLE
Update FAQ and Admin guide

### PR DESCRIPTION
**Pull request by stevenGravy**
Added Multiplex faq entry and within reverse tunneling on admin-guide, Corrected enhanced logging comments in teleport configuration

### CODE REVIEW

File: docs/4.2/admin-guide.md [hunk 1]
```diff
@@ -1016,6 +1017,9 @@ and 3024 on the proxy. Port 3080 is used to initially fetch the credentials (SSH
 and for discovery (where is the reverse tunnel running, in this case 3024). Port 3024 is used to
 establish a connection to the Auth Server through the proxy.
 
+To enable multiplexing so only one port is used simply set the `tunnel_public_addr` and `tunnel_listen_addr` the same as the `public_addr`
```
Comment (webvictim) [inline docs/4.2/admin-guide.md:1020]: is used simply -> is used, simply (add comma).

Also, I don't think this is actually what enables multiplexing. Explicitly, `listen_addr` and `tunnel_listen_addr` have to be set to the same port. There is also a log message printed to do with multiplexing when Teleport is started in this mode, which we might want to highlight so people can know they're doing it right.

File: docs/4.2/faq.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ it has been fixed with Teleport 4.0.   Once you've upgraded your Teleport Cluste
 option `--auth-server` to point to web proxy address (this would be `public_addr` and `web_listen_addr` 
 in file configuration). As defined in [Adding a node located behind NAT - Teleport Node Tunneling](admin-guide.md#adding-a-node-located-behind-nat)
 
+## Can nodes use a single port for reverse tunnels?
+
+Yes, Teleport supports multiplexing on a single port.  The tunnel_public_addr and tunnel_listen_addr can be set to the same port as the web address setting. Teleport will automatically use multiplexing with that configuration.
```
Comment (webvictim) [inline docs/4.2/faq.md:40]: As above, `listen_addr` and `tunnel_listen_addr` need to be set to the same port.

File: docs/4.2/admin-guide.md [hunk 3]
```diff
@@ -426,12 +426,13 @@ ssh_service:
...
-    # Enhanced Session Recording was introduced with Teleport 4.2. For more details 
-    # see 
+    # Enhanced Session Recording was introduced with Teleport 4.2.   To enable 
+    # enhanced sessions recording the machine must have Linux Kernel (4.18+)  and BCC tools installed. For more details 
+    # see https://gravitational.com/teleport/docs/features/enhanced_session_recording/ 
     enhanced_recording:
        # Enable or disable enhanced auditing for this node. Default value: 
        # false.
-       enabled: true
+       enabled: false
```
[the remark this answers is not in the document]
Comment (stevenGravy) [reply docs/4.2/admin-guide.md:435]: @webvictim  We don't have pam enabled in this example which is the default, same as this setting to false.  The separate  pam  and the enhanced logging have a enabled example.  I'd say either both are turned on or both are off.  My concern with having both on is that you have to do more settings in the environment for those then just copying in the yaml.

File: docs/4.2/admin-guide.md (hunk 1 above)
Comment (stevenGravy) [reply docs/4.2/admin-guide.md:1020]: I don't see the log message in 4.3 alpha.  I did confirm it works in 4.3 and 4.2.

Comment (stevenGravy) [reply docs/4.2/admin-guide.md]: changed the language.

Comment (stevenGravy) [reply docs/4.2/admin-guide.md]: Found the output if set to debug

**Comment by stevenGravy** [comment]
should be good @webvictim !

**Comment by benarent** [comment]
Looks good, can you also create another PR for 4.3 docs `docs/4.3-base` so this doesn't get lost.

**Pull request closed without merging**